src/AnnotationReader/ReaderException.php

```diff
@@ -9,7 +11,6 @@ class ReaderException extends \Exception
 {
     /**
      * @param string $message
-     * @param mixed  $item
      */
     public function __construct($message, $item)
```

Comment:
@HypeMC Add type-hints to constructor arguments directly:

```php
public function __construct(string $message, mixed $item)
```

Reply:
I've already started doing code improvements on a different branch, let's keep at as is on this one.

Reply:
In case you're curious https://github.com/sofascore/purgatory/commit/1c51784282f5783e594f9d43ca9899dd4cee3d74